### ISSUE TITLE
self-define loss function

### ISSUE THREAD

**Issue by LawrenceXH** [MachineLP/Spark-#2]
Thanks for your work.Can you explain how to construct a self-define loss function like mae or mape in your  sparkxgb.If its alike in use in python+xgboost .
